### PR TITLE
MEPTS-49 Implement TX_NEW changes

### PR DESCRIPTION
**Pull request by ynurmahomed**

### CODE REVIEW

**Comment by nthfloor** [comment]
Just waiting for [this PR](https://github.com/esaude/openmrs-module-eptsreports/pull/10) to be merged before merging

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/HivCohortQueries.java [hunk 1]
```diff
@@ -75,4 +79,10 @@ public CohortDefinition getPatientsViralLoadWithin12Months() {
 		    hivMetadata.getHivViralLoadConcept().getConceptId()));
 		return sql;
 	}
+
```
[the remark this answers is not in the document]
Comment (ynurmahomed) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/HivCohortQueries.java]: Comment added

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BaseQueries.java [hunk 2]
```diff
@@ -24,8 +24,7 @@
 	// they map to concept_id=1369 - TRANSFER FROM OTHER FACILITY
 	// TODO: Query needs to be refactored
 	public static String getBaseCohortQuery(Map<String, String> parameters) {
-		String query = "select patient_id from "
-		        + "(select p.patient_id from patient p join encounter e on e.patient_id=p.patient_id join person pr on pr.person_id = p.patient_id "
+		String query = "select p.patient_id from patient p join encounter e on e.patient_id=p.patient_id join person pr on pr.person_id = p.patient_id "
 		        + "where e.voided=0 and p.voided=0 and e.encounter_type in (%s) and e.encounter_datetime<=:endDate and e.location_id = :location and pr.birthdate is not null "
```
Comment (nthfloor) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BaseQueries.java]: @ynurmahomed this query was updated in this commit: 7e326c4773aab0837e252ac8128d5ce0398d62be

Please pull that change in.

Comment (ynurmahomed) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BaseQueries.java]: @nthfloor done

**Comment by nthfloor** [comment]
@ynurmahomed please resolve these conflicts so we can merge this into master.

**Comment by ynurmahomed** [comment]
@nthfloor conflicts resolved

**Pull request merged**